noodles/display/dumb_term.py

```diff
@@ -50,21 +53,20 @@ def report(self):
             self.out << "[ERROR!]\n\n"
 
             for job, e in self.errors:
-                msg = 'ERROR '
```

Comment:
No need to manually prefix `ERROR` if it's already included via the logging level.